sway-lsp/src/core/token_map.rs

```diff
@@ -17,7 +17,7 @@ pub struct TokenMap(DashMap<TokenIdent, Token>);
 impl TokenMap {
     /// Create a new token map.
     pub fn new() -> TokenMap {
-        TokenMap(DashMap::new())
+        TokenMap(DashMap::with_capacity(2048))
```

Comment:
How did you determine the value for the capacity?

Reply:
I just chose a large power of two number that seemed like it would be a reasonable default. We would likely also be fine using 1024